chapters/fr/_toctree.yml

```
@@ -3,16 +3,36 @@
   - local: chapter0/1
     title: Introduction
 
+ title: 2. Utilisation de 🤗 Transformers
```

I think we need a `-` here :)

```suggestion
- title: 2. Utilisation de 🤗 Transformers
```

---

Done

---

Thank you!